clusters/hive/hypershift.yaml

```diff
@@ -113,3 +113,85 @@ subjects:
   - kind: ServiceAccount
     namespace: ci
     name: hypershift-workload
+---
+apiVersion: machine.openshift.io/v1beta1
+kind: MachineSet
+metadata:
+  name: hypershift-worker
+  namespace: openshift-machine-api
+spec:
+  replicas: 1
```

Comment:
This should not be set if we have a `MachineAutoscaler` targeting this machine set.